cmd/nri-kubernetes/main.go

```diff
@@ -97,7 +107,7 @@ func main() {
 	}
 }
 
-func runScrapers(c config.Mock, ksmScraper *ksm.Scraper, kubeletScraper *kubelet.Scraper, i *integration.Integration, clients *clusterClients) error {
+func runScrapers(c config.Mock, ksmScraper *ksm.Scraper, kubeletScraper *kubelet.Scraper, controlplaneScraper *controlplane.Scraper, i *integration.Integration, clients *clusterClients) error {
```

Comment:
just added extra parameter, but we discussed to refactor this in the future to have a scraper interface array but I didn't want to add an extra refactor to this PR